### PR TITLE
SorrTask64: Move GitHub_ZenHub_workflow doc to md

### PR DESCRIPTION
**Pull request by DanilYachmenev**
#64 

Follow-up from Asana task to add some documentation about how to report an Issue

Found and moved https://docs.google.com/document/d/1M8weNOzMnS7vdMPzEyQ4g7OsCUXXsCn60qzPWsA7SlQ/edit#heading=h.vag5u04x37u3

However, it definitely has some intersections with https://github.com/sorrentum/sorrentum/pull/201 so we probably combine them in 1 doc

But at first I'd merge their md versions

### CODE REVIEW

File: docs/GitHub_ZenHub_workflows.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
+# Introduction
```
Comment (gpsaggese) [inline docs/GitHub_ZenHub_workflows.md:1]: When you convert a gdoc can you pls
1) add a new entry to the README https://github.com/sorrentum/sorrentum#documents point me to the new markdown file, with a short description of what the new markdown contains
2) strike out the entire Gdoc and add a pointer to the markdown (then after a bit we can delete the Gdoc)

File: docs/GitHub_ZenHub_workflows.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+## PR labels 
+
+### `# TODO(Dan): Update for Sorrentum.`
+
+- `PR_for_authors`
+    - There are changes to be addressed by an author of a PR
+- `PR_for_reviewers`
+    - PR is ready for review by RPs
+- `PR_for_integrators`
+    - PR is ready for the final round of review by GP, i.e. close to merge
```
Comment (DanilYachmenev) [inline docs/GitHub_ZenHub_workflows.md:158]: Should we  just remove it for Sorrentum?

File: docs/GitHub_ZenHub_workflows.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+  - What the problem is
+  - Why the outcome is different from what you expected
+- Use check boxes for "small" actions that need to be tracked in the issue (not worth their own bug)
+    - An issue should be closed only after all the checkboxes have been addressed
+    - `# TODO(Dan): Add a pic with example as in Gdoc.`
+- We use the `FYI @...` syntax to add "watchers"
+  - E.g., `FYI @cryptomtc` so that he receives notifications for this issue
+  - Authors and assignees receive all the emails in any case
+  - In general everybody should be subscribed to receiving all the notifications and you can quickly go through them to know what's happening around you
+  - `# TODO(Dan): Add a pic with example as in Gdoc.`
```
Comment (DanilYachmenev) [inline docs/GitHub_ZenHub_workflows.md:114]: Do we need to add a pictures to`.md` files?

File: docs/GitHub_ZenHub_workflows.md (hunk 2 above)
Comment (DanilYachmenev) [reply docs/GitHub_ZenHub_workflows.md]: Added respective labels to Sorrentum as well so all is in sync

**Pull request merged**